fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1039,6 +1041,32 @@ public void testIngestOnSubtree() throws Exception {
 
     }
 
+    /**
+     * Ensure that the objects cannot be pairtree child resources
+     *
+     */
+    @Test
+    public void testIngestOnPairtree() throws Exception {
+        HttpPost method = postObjMethod("");
+        HttpResponse response = client.execute(method);
+
+        //  Following the approach undertaken for FedoraExportIT#shouldRoundTripOnePairtree
+        final String objName = response.getFirstHeader("Location").getValue();
+        final String pairtreeName = objName.substring(serverAddress.length(), objName.lastIndexOf('/'));
```

Comment:
This looks a lot better, but I'm still a bit weirded out by the slash-searching. Is there no RDF relationship published between a resource and its pair-tree "container"?